src/app/core/components/terms-pages/terms-pages.component.ts

```diff
@@ -0,0 +1,127 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-terms-pages',
+  templateUrl: './terms-pages.component.html',
+  styleUrls: ['./terms-pages.component.scss'],
+})
+export class TermsPagesComponent implements OnInit {
+  constructor(private router: Router) {}
+
+  data;
+
+  ngOnInit(): void {
```

Comment:
сделать импорт данных с отдельного файла и сделать следущим образом:

```suggestion
  ngOnInit(): void {
     this.initData();
  }
  
  private initData(): void {
      const { url } = this.router;
      
      this.data = url === '/terms' ? componentData.termsData : componentData.privacyData;
  }
```